Language Adjustment - Ducks

CorruptOmega opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Ducks now speak and understand Duck, instead of Sylvan. Why was a Duck speaking like a plant?

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
Slight fix to the duck. 

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
:cl: CorruptOmega
- changed: Ducks now speak and understand Duck, instead of Sylvan.

<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.
-->

CorruptOmega committed:
Adjusting Languages

CorruptOmega committed:
Removed speaking all languages from pAI and Cyborgs

CorruptOmega committed:
Merge branch 'Languages-Adjustment' of https://github.com/CorruptOmega/space-station-14-Starlight into Languages-Adjustment

Happyrobot33 commented on Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml:
It makes no sense for borgs to be able to understand full animal languages

CorruptOmega replied:
I was thinking it would be that AI would automatically upload Cyborgs/pAI's with understanding all languages to interact with everything on station. Just not being able to speak the languages was due to limited processing.

Though I guess since the animal wouldn't understand common without cognizine anyways... it's kind of moot for cyborg to be able to understand them but not the other way around.

Happyrobot33 commented on Resources/Prototypes/Entities/Mobs/NPCs/moproach.yml:
Mothroaches are considered station pets, and all station pets are given common as a understandable language

CorruptOmega replied:
Ah. I see.

CorruptOmega committed:
Update moproach.yml

CorruptOmega committed:
Update base_borg_chassis.yml

Removed all animal languages EXCEPT for Ancestor, so Cyborgs can understand Pun Pun.

CorruptOmega committed:
Update pai.yml

CorruptOmega committed:
Update base_borgi_chassis.yml

CorruptOmega commented:
Updated Cyborgs, pAI's and Borgis to no longer understand most of the animals. Left in Ancestor so that they can communicate with Pun Pun.

ShadesMars commented on Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml:
Bogs should understand but not speak.

(In the future il allow it with a module pehaps)

ShadesMars commented on Resources/Prototypes/Entities/Objects/Fun/pai.yml:
```diff
@@ -28,6 +28,7 @@
     - VoxPidgin
     - Scratch
     - SolCommon
+    - Ancestor
```
```suggestion
```

ShadesMars commented on Resources/Prototypes/Entities/Mobs/Player/silicon.yml:
```diff
@@ -330,6 +348,12 @@
+    - Dog
+    - Chicken
+    - Duck
+    - Mouse
```
```suggestion
```

ShadesMars commented on Resources/Prototypes/Entities/Mobs/Player/silicon.yml:
```diff
@@ -314,6 +314,24 @@
     speaks:
     - GalacticCommon
     - Machine
+    - Ancestor
```
```suggestion
```

ShadesMars commented on Resources/Prototypes/Entities/Mobs/Player/silicon.yml:
```diff
@@ -314,6 +314,24 @@
+    - Chicken
+    - Chittin
+    - Dog
+    - Duck
```
```suggestion
    - Chittin
```

ShadesMars commented on Resources/Prototypes/Entities/Mobs/Player/silicon.yml:
```diff
@@ -314,6 +314,24 @@
+    - Draconic
+    - Marish
+    - Moffic
+    - Mouse
```
```suggestion
```

ShadesMars commented on Resources/Prototypes/_StarLight/Entities/Mobs/Cyborgs/base_borgi_chassis.yml:
```diff
@@ -204,7 +203,8 @@
       - Scratch
-      - Dog
+      - SolCommon
+      - Ancestor
```
```suggestion
```

ShadesMars reviewed (changes requested):
No one should understand/speak animals.

AI to speak all languages is fine, for bogs no unless in the future changed by a module.

CorruptOmega committed:
Update Resources/Prototypes/Entities/Objects/Fun/pai.yml

Co-authored-by: FoxxoTrystan <45297731+FoxxoTrystan@users.noreply.github.com>

CorruptOmega committed:
Update Resources/Prototypes/Entities/Mobs/Player/silicon.yml

Co-authored-by: FoxxoTrystan <45297731+FoxxoTrystan@users.noreply.github.com>

CorruptOmega committed:
Update Resources/Prototypes/Entities/Mobs/Player/silicon.yml

Co-authored-by: FoxxoTrystan <45297731+FoxxoTrystan@users.noreply.github.com>

CorruptOmega committed:
Update Resources/Prototypes/Entities/Mobs/Player/silicon.yml

Co-authored-by: FoxxoTrystan <45297731+FoxxoTrystan@users.noreply.github.com>

CorruptOmega committed:
Update Resources/Prototypes/Entities/Mobs/Player/silicon.yml

Co-authored-by: FoxxoTrystan <45297731+FoxxoTrystan@users.noreply.github.com>

CorruptOmega committed:
Update Resources/Prototypes/_StarLight/Entities/Mobs/Cyborgs/base_borgi_chassis.yml

Co-authored-by: FoxxoTrystan <45297731+FoxxoTrystan@users.noreply.github.com>

ShadesMars commented on Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml:
This will make a conflict merge with #1454

ShadesMars commented on Resources/Prototypes/Entities/Mobs/Player/silicon.yml:
This will make a conflict merge with #1454

ShadesMars commented on Resources/Prototypes/_StarLight/Entities/Mobs/Cyborgs/base_borgi_chassis.yml:
This will make a conflict merge with #1454

ShadesMars reviewed (commented):
Prepare for a bunch of conflict merge so il not approve this yet, however.

Duck changes are good and this may need to be checked after #1454 but AI should be able to speak all "Species" Languages.

CorruptOmega commented:
Yeah. I'm gonna need to figure out how all that Ai stuff is getting sorted out once 1454 is merged...

CorruptOmega commented:
> Prepare for a bunch of conflict merge so il not approve this yet, however.
> 
> Duck changes are good and this may need to be checked after #1454 but AI should be able to speak all "Species" Languages.

Could just have walks add the AI speaking to https://github.com/ss14Starlight/space-station-14/pull/1454 and I can further whittle this down to only affecting duck.

ShadesMars commented:
good idea

CorruptOmega committed:
Removed changes, as another PR is changing them.

CorruptOmega committed:
Removed changes, as another PR is changing them.

CorruptOmega committed:
Removed changes, as another PR is changing them.

CorruptOmega commented:
Hmm. I have a feeling like deleting files in this PR is not what I'm supposed to do. Fixing when I get home.

CorruptOmega committed:
Revert "Removed changes, as another PR is changing them."

This reverts commit 235bd6abfa54bcbccd0ecd0f3f4d4a9e3a0c9801.

CorruptOmega committed:
Revert "Removed changes, as another PR is changing them."

This reverts commit c45549fe3f0d084bd1dcd2a7f77e2989de2d4224.

CorruptOmega committed:
Revert "Removed changes, as another PR is changing them."

This reverts commit 6f9036de33ccfa4f8aa9757f2f1861268013ecb2.

CorruptOmega commented:
Due to the amount of stuff just changed overall and removed, I'll just make a new pull request with just the duck change.
